### PR TITLE
Request-based rma testall fix

### PR DESCRIPTION
**Pull request by minsii**

### CODE REVIEW

**Comment by minsii** [comment]
test:jenkins/ch4/most

**Comment by minsii** [comment]
test:jenkins/ch3/all

**Comment by minsii** [comment]
@hajimefu : would you mind review this PR ?

**Comment by hajimefu** [comment]
Sure, I'll do it.

**Comment by raffenet** [comment]
@minsii hey, the test triggers worked this time!

File: test/mpi/rma/rget-testall.c [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    MPI_Win_unlock_all(win);
+    MPI_Barrier(MPI_COMM_WORLD);
+
+    MPI_Win_free(&win);
+
+    MTest_Finalize(errs);
+    MPI_Finalize();
+}
```
Comment (hajimefu) [inline test/mpi/rma/rget-testall.c]: Would be nice if `main` returns `errs != 0` (so a shell script can easily see the result without scraping "No Errors" string). See #2640.

Comment (minsii) [reply test/mpi/rma/rget-testall.c]: Good catch !

**Comment by minsii** [comment]
test:jenkins/ch4/most
test:jenkins/ch3/all

**Comment by minsii** [comment]
test:jenkins/ch3/ofi

**Comment by minsii** [comment]
test:jenkins/ch3/ofi

**Comment by minsii** [comment]
test:jenkins/ch4/ofi

**Comment by raffenet** [comment]
This looks good to merge. The only failures present can be ignored.

**Pull request merged**